apps/flint/src/commands/ping.ts

```diff
@@ -0,0 +1,13 @@
+import { SlashCommand } from "interaction-kit";
+
+export default new SlashCommand({
```

Comment:
Just gonna add a simple ping command for now, but we can/should add things for all the ikit functionality (buttons, selects, arg types, etc)